docs/guide/data_types/timeseries.rst

```diff
@@ -441,30 +306,103 @@ console view: ::
+    >>> from astropy.time import Time
+    >>> import astropy.units as u
+    >>> from astropy.table import Table
+
+    >>> base = datetime.datetime.today()
+    >>> times = [base - datetime.timedelta(minutes=x) for x in range(24*60, 0, -1)]
+    >>> intensity = u.Quantity(np.sin(np.arange(0, 12 * np.pi, ((12 * np.pi) / (24*60)))), u.W/u.m**2)
+    >>> tbl_meta = {'t_key':'t_value'}
+    >>> table = Table([times, intensity], names=['time', 'intensity'], meta=tbl_meta)
+    >>> table.add_index('time')
+    >>> ts_table = ts.TimeSeries(table)
+
+Note that due to the properties of the `~astropy.time.Time` object, this will be a mixin column which since it is a single object, limits the versatility of the `~astropy.table.Table` a little.
+For more on mixin columns see the `Astropy docs <https://docs.astropy.org/en/stable/table/mixin_columns.html>`_.
+The units will be taken from the table quantities for each column, the metadata will simply be the table.meta dictionary.
+You can also explicitly add metadata and units, these will be added to the relevant dictionaries using the dictionary update method, with the explicit user-given values taking precedence.
+
+    >>> from sunpy.util.metadata import MetaDict
+    >>> from collections import OrderedDict
+    >>> import astropy.units as u
+
+    >>> meta = MetaDict({'key':'value'})
+    >>> units = OrderedDict([('intensity', u.W/u.m**2)])
+    >>> ts_table = ts.TimeSeries(table, meta, units)
 
+.. _ts-metadata:
 
 6. A Detailed Look at the Metadata
 ==================================
 
-TimeSeries store metadata in a `~sunpy.timeseries.TimeSeriesMetaData`
-object, this object is designed to be able to store multiple basic `~sunpy.util.metadata.MetaDict`
-(case-insensitive ordered dictionary) objects and able to identify the relevant
-metadata for a given cell in the data. This enables a single TimeSeries to be
-created by combining/concatenating multiple TimeSeries source files together
-into one and to keep a reliable track of all the metadata relevant to each cell,
-column or row.  The metadata can be accessed by: ::
+TimeSeries store metadata in a `~sunpy.timeseries.TimeSeriesMetaData` object, this object is designed to be able to store multiple basic `~sunpy.util.metadata.MetaDict` (case-insensitive ordered dictionary) objects and able to identify the relevant metadata for a given cell in the data.
+This enables a single TimeSeries to be created by combining/concatenating multiple TimeSeries source files together into one and to keep a reliable track of all the metadata relevant to each cell, column or row.
+The metadata can be accessed by: ::
 
     >>> meta = my_timeseries.meta
 
-You can easily get an overview of the metadata, this will show you a basic
-representation of the metadata entries that are relevant to this TimeSeries. ::
+You can easily get an overview of the metadata, this will show you a basic representation of the metadata entries that are relevant to this TimeSeries. ::
```

Comment:
```suggestion
You can easily get an overview of the metadata, this will show you a basic representation of the metadata entries that are relevant to this TimeSeries.

.. codeblock:: python
```